### PR TITLE
[tests] split next-local tests into chunks

### PR DESCRIPTION
**Pull request by EndangeredMassa**
The next-local e2e tests where split between middleware (fast) and all other (slow) tests. This meant there was not a good balance of amount of time spent across these test files, making overall test run times as slow as "all other tests" running.

This PR rebalances the split of tests across 2 files. The new run time is ~9-10m.

### CODE REVIEW

File: packages/next/test/integration/integration-1.test.js [hunk 1]
```diff
@@ -2,15 +2,54 @@ process.env.NEXT_TELEMETRY_DISABLED = '1';
 
 const path = require('path');
 const fs = require('fs-extra');
-const builder = require('../../');
+const builder = require('../../dist');
```
Comment (styfle) [inline packages/next/test/integration/integration-1.test.js]: Was this intentional?

File: packages/next/test/integration/integration-1.test.js
Comment (styfle) [inline packages/next/test/integration/integration-1.test.js:1]: Lets make sure this file also sets `process.env.NEXT_TELEMETRY_DISABLED = '1';`

Comment (EndangeredMassa) [reply packages/next/test/integration/integration-1.test.js:1]: It does, but it was tucked into a single line of github line folding

File: packages/next/test/integration/integration-1.test.js (hunk 1 above)
Comment (EndangeredMassa) [reply packages/next/test/integration/integration-1.test.js]: It was not. Updated. Thanks!
[`3d0e1ce` (#9774)](https://github.com/vercel/vercel/pull/9774/commits/3d0e1cea2e7fd171aa865411ed1ffb8f10b43d98)

**Review by cb1kenobi — APPROVED** [review]
LGTM

**Pull request merged**